challenges/palindrome-detector.js

```diff
@@ -21,3 +21,23 @@
 */
 
 // YOUR CODE HERE
+
+let isPalindrome = function(string){
+  let letters = string.split('');
+  let length = letters.length; 
+  for (i=0;i<string.length;i++){ 
+    if (letters[i] != letters[length-1]){
+      return false; 
+
+    }
+    else{
+      return true; 
```

Comment:
It looks like this will always return true if the first two characters match--make sure to test your code against multiple cases! (So for example, if you have the string "delivered" it'll return true because the first/last characters match, even though the string overall isn't a palindrome.)